src/client/dfuse/dfuse.h

```diff
@@ -474,6 +481,10 @@ struct dfuse_pool {
 	ACTION(OPEN)                                                                               \
 	ACTION(PRE_READ)                                                                           \
 	ACTION(READ)                                                                               \
+	ACTION(READ_EOF_M)                                                                         \
+	ACTION(READ_CON)                                                                           \
+	ACTION(READ_BUCKET)                                                                        \
+	ACTION(READ_BUCKET_M)                                                                      \
```

Comment:
Extra metrics in dfuse is a good thing but I'm thinking of removing these from this PR.  They were certainly useful to detect/different code-paths but I think I'd rather extend these outside of a bug-fix PR.